docs/_layouts/ds6/default.html

```diff
@@ -1,17 +1,67 @@
 <!doctype html>
 <html lang="en">
     <head>
-        <title>eBay skin DS6</title>
+        <title>eBay Skin — for DS6</title>
         <meta name="viewport" content="width=device-width, initial-scale=1"/>
         <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
         <link type="text/css" rel="stylesheet" href="../static/ds6/skin.css"/>
         <link type="text/css" rel="stylesheet" href="../static/ds6/docs.css"/>
     </head>
     <body>
+        <header>
+            <h1>
+                <span class="inline-svg" aria-hidden="true" role="img">
```

Comment:
We want a screen reader to read this as "eBay" image. So remove aria-hidden, and add aria-label="eBay". Remove aria-labelledby from the svg tag.

Reply:
👍 Will do.